build.sbt

```diff
@@ -56,126 +56,44 @@ lazy val routeguide = project
 /////   SEED   /////
 ////////////////////
 
-//// Shared Modules ////
-
 lazy val `seed-config` = project
-  .in(file("seed/shared/modules/config"))
-  .settings(exampleSeedConfigSettings)
-  .disablePlugins(SrcGenPlugin)
-
-////     Shared     ////
-
-lazy val allSharedModules: ProjectReference = `seed-config`
-
-lazy val `seed-shared` = project
   .in(file("seed/shared"))
-  .aggregate(allSharedModules)
-  .disablePlugins(SrcGenPlugin)
-
-//////////////////////////
-////  Server Modules  ////
-//////////////////////////
-
-lazy val `seed-server-common` = project
-  .in(file("seed/server/modules/common"))
-  .disablePlugins(SrcGenPlugin)
-
-lazy val `seed-server-protocol-avro` = project
-  .in(file("seed/server/modules/protocol_avro"))
-  .settings(libraryDependencies ++= Seq(mu("mu-rpc-service")))
-  .disablePlugins(SrcGenPlugin)
-
-lazy val `seed-server-protocol-proto` = project
-  .in(file("seed/server/modules/protocol_proto"))
-  .settings(libraryDependencies ++= Seq(mu("mu-rpc-fs2")))
-  .disablePlugins(SrcGenPlugin)
-
-lazy val `seed-server-process` = project
-  .in(file("seed/server/modules/process"))
-  .settings(exampleSeedLogSettings)
-  .dependsOn(`seed-server-common`, `seed-server-protocol-avro`, `seed-server-protocol-proto`)
+  .settings(exampleSeedConfigSettings)
   .disablePlugins(SrcGenPlugin)
 
-lazy val `seed-server-app` = project
-  .in(file("seed/server/modules/app"))
-  .settings(libraryDependencies ++= Seq(mu("mu-rpc-server")))
-  .dependsOn(`seed-server-process`, `seed-config`)
+lazy val `seed-protocol` = project
```

Comment:
This is the most potentially controversial change I made IMO; since the seed-server and the seed-client rely on the avro and protobuf protocols, I ended up making a new directory for all of the protocol code so this example looked more like the standard that we're setting on other examples.  LMK if there's a way you'd prefer instead!